app/controllers/miq_ae_class_controller.rb

```diff
@@ -2532,16 +2533,17 @@ def domain_overrides
     end
   end
 
-  def get_session_data
-    @layout     = "miq_ae_class"
-    @title      = _("Datastore")
-    @lastaction = session[:aeclass_lastaction]
-    @edit       = session[:edit]
+  def title
+    @title = _("Datastore")
+  end
+
+  def session_key_prefix
+    @layout = "miq_ae_class"
   end
 
   def set_session_data
-    session[:aeclass_lastaction] = @lastaction
-    session[:edit]               = @edit
+    super
+    session[:edit] = @edit
```

Comment:
This line is not needed - done in `set_global_session_data` - so you can remove the whole `set_session_data` method.